src/classes/neutronWeights.cpp

```diff
@@ -259,10 +232,16 @@ void NeutronWeights::createFromIsotopologues(const std::vector<std::shared_ptr<A
         {
             const auto *top = isoWeight.isotopologue();
 
-            // Loop over Atoms in the Species, searching for the AtomType/Isotope entry in the isotopes list of the Isotopologue
+            // Loop over Atoms in the Species, searching for the AtomType/Isotope entry in the isotopes list of the
+            // Isotopologue
             for (const auto &i : topes.species()->atoms())
+            {
+                if (!i.isPresence(SpeciesAtom::Presence::Physical))
+                    continue;
+
                 atomTypes_.addIsotope(i.atomType(), top->atomTypeIsotope(i.atomType()),
                                       isoWeight.weight() * topes.speciesPopulation());
+            }
```

Comment:
```suggestion
            for (const auto &i : topes.species()->atoms())
                if (i.isPresence(SpeciesAtom::Presence::Physical))
                    atomTypes_.addIsotope(i.atomType(), top->atomTypeIsotope(i.atomType()),
                                      isoWeight.weight() * topes.speciesPopulation());
```

Possible rewrite.  Pick whichever you prefer

Reply:
Not sure why I had written it that way before!